### PR TITLE
LPD-23761 Sanitize portletId with method getJsSafePortletId same as i…

### PR DESCRIPTION
**Pull request by georgel-pop-lr**
…s done in RuntimeTag, so it can be correctly identified

For steps please read: https://liferay.atlassian.net/issues/LPD-23761

When we render the embedded portlets, we sanitize the instance id here https://github.com/liferay/liferay-portal/blob/983e9163db9f359a6d468f62165ceb53f14b0cde/util-taglib/src/com/liferay/taglib/portletext/RuntimeTag.java#L174 this calls this method which eliminates all dashes, spaces or dots
https://github.com/liferay/liferay-portal-ee/blob/d3f5c54e4523a1b57f3a6364ef268e9ee772c906/util-java/src/com/liferay/util/JS.java#L92

Note: This on 7.2 was working as we collected the portletIds from portletpreferences table (see [LayoutCopyHelperImpl.java#L361](https://github.com/liferay/liferay-portal-ee/blob/d3f5c54e4523a1b57f3a6364ef268e9ee772c906/modules/apps/layout/layout-impl/src/main/java/com/liferay/layout/internal/util/LayoutCopyHelperImpl.java#L361)), and we copied them to the specific layout plid number

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

File: modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/processor/PortletRegistryImpl.java [hunk 1]
```diff
@@ -257,6 +258,9 @@ private Document _getDocument(String html) {
 	@Reference
 	private JSONFactory _jsonFactory;
 
+	@Reference
+	private PortalUtil _portalUtil;
```
Comment (ealonso) [inline modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/processor/PortletRegistryImpl.java:262]: @georgel-pop-lr it should be `private  Portal _portal;` you should use the interface and not the util class.

**Comment by ealonso** [comment]
Hey @georgel-pop-lr,

I think tests failures are related with your changes, Could you please take a look?

**Comment by georgel-pop-lr** [comment]
ci:report:2562115527

**Comment by georgel-pop-lr** [comment]
ci:test:sf

**Comment by georgel-pop-lr** [comment]
ci:test:relevant

**Pull request closed without merging**